docs/journey/day1/README.md

```diff
@@ -41,6 +41,12 @@ The `kam bootstrap` [command](../../commands/kam_bootstrap.md) also provides an
 
 In the event of using a self-hosted _GitHub Enterprise_ or _GitLab Community/Enterprise Edition_ if the driver name isn't evident from the repository URL, use the `--private-repo-driver` flag to select _github_ or _gitlab_.
 
+* When a token is provided in command flag `--git-host-access-token`, the token will be stored securely in keyring. The hostname of the URL (e.g. github.com) will be used to stored the token by keyring. The provided token will be used.
+
+* When a token is not provided in command flag `--git-host-access-token`, the token is retrieved by keyring using the hostname of the URL. If no token is not found, a token is retrieved from env var. What is the format of the env var? If no token is not found in the env var, the command will fail.
+
+* When a token is not provided in command flag and the token is not present in the keyring, the webhook cmd will look for the access token in an environment variable with the syntax HOSTNAME_TOKEN (e.g. GITHUB_COM_TOKEN).
```

Comment:
This is not strictly applied to webhook command.  It also applies to bootsrap command.   

You should clearly spell out how to derive an environment name from an URL.  Token is read from environment variable of environment name derived from git URL.
* hostname of the URL is extracted with all "." replaced by "_", and converted to all upper cases.  <hostname_string>
* hostname is appended by `_TOKEN` 
* the format of the enviornment variable is <hostname_string>_TOKEN